dataplanes/vpp/build/Dockerfile.build

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:bionic as govppbuilder
+ARG DEBIAN_FRONTEND=noninteractive
+ARG REPO=release
+RUN apt-get update
+RUN apt-get -y install curl
+RUN curl -s https://packagecloud.io/install/repositories/fdio/${REPO}/script.deb.sh |  bash
+RUN apt-get -y install vpp vpp-plugins vpp-dev golang git curl
+ENV GOPATH="/root/go"
+RUN mkdir -p $GOPATH/bin
+RUN mkdir -p $GOPATH/pkg
+RUN mkdir -p $GOPATH/src
+RUN curl -s https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+RUN mkdir -p $GOPATH/src/workspace/ligato/vpp
+ARG CACHE_DATE
```

Comment:
Ah, the tricky hack we discussed! ;)

Reply:
Probably it will need to be removed if VPP need to be integrated into CI image building/pushing.